### ISSUE TITLE
CSSCombBear

### ISSUE THREAD

**Issue by sils** [coala/coala-bears#28]
_From @sils1297 on February 10, 2016 16:29_

http://csscomb.com

_Copied from original issue: coala-analyzer/coala#1398_

**Comment by sils1297** [comment]
_From @AbdealiJK on February 16, 2016 6:35_

We *could* write it to a temporary file, let csscomb fix that and read it back - but that's really really ugly.
i suggest we wait until https://github.com/csscomb/csscomb.js/issues/451 is done.

Blocked by https://github.com/csscomb/csscomb.js/issues/451

**Comment by yash-nisar** [comment]
Duplicate of https://github.com/coala/coala-bears/issues/634 so can be closed. :)

**Comment by yash-nisar** [comment]
@meetmangukiya Can you close this one too ? :stuck_out_tongue:

**Comment by yash-nisar** [comment]
@meetmangukiya Sorry for making you close the issue, but this issue is rather important and needs to be reopened. The title is pretty misleading and needs to be improved to something like `Parsing output from csscomb` and we should add an upstream label to it.
